assignments/#10.1.py

```diff
@@ -0,0 +1,17 @@
+
+#10.1 Split large number into hundreds, tens and units.
+
+
+print('This program split given large number into hundreds, tens and units.')
+
+
+num = float(raw_input('Enter a number: '))

```

Comment:
try not to use console input/output. If you need to implement something use constants or/and functions and testcases. I'll explain more later.